### PR TITLE
Updated parser version for Scala version 2.12

### PR DESCRIPTION
**Pull request by upx86**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,11 +55,11 @@
     </distributionManagement>
 
     <dependencies>
-        <!-- https://mvnrepository.com/artifact/org.scala-lang.modules/scala-parser-combinators_2.11 -->
+        <!-- https://mvnrepository.com/artifact/org.scala-lang.modules/scala-parser-combinators -->
         <dependency>
             <groupId>org.scala-lang.modules</groupId>
-            <artifactId>scala-parser-combinators_2.11</artifactId>
-            <version>1.0.6</version>
+            <artifactId>scala-parser-combinators_2.12</artifactId>
```
Comment (facaiy) [inline pom.xml:61]: Thank you. Would you mind explaining why to use 2.12 here? I suppose the package is still compiled by 2.11, see

> <scala.version>2.11.5</scala.version>
> <scala.compat.version>2.11</scala.compat.version>

Comment (upx86) [reply pom.xml:61]: My bad, I override it with SBT in my solution.

**Review by upx86 — COMMENTED** [review]
Updated Scala compiler and test version

**Comment by facaiy** [comment]
Sorry for the delay, thanks.

**Pull request merged**